astropy/coordinates/builtin_frames/itrs_observed_transforms.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+import erfa
+from astropy import units as u
+from astropy.coordinates.matrix_utilities import rotation_matrix, matrix_transpose
+from astropy.coordinates.baseframe import frame_transform_graph
+from astropy.coordinates.transformations import FunctionTransformWithFiniteDifference
+from astropy.coordinates.representation import CartesianRepresentation
+from .altaz import AltAz
+from .hadec import HADec
+from .itrs import ITRS
+from .utils import PIOVER2
+
+# Minimum cos(alt) and sin(alt) for refraction purposes
+CELMIN = 1e-6
+SELMIN = 0.05
+
+
+def itrs_to_altaz_mat(lon, lat):
+    # form ITRS to AltAz matrix
+    elat = lat.to_value(u.radian)
+    elong = lon.to_value(u.radian)
+    # AltAz frame is left handed
+    minus_x = np.eye(3)
+    minus_x[0][0] = -1.0
+    mat = (minus_x
+           @ rotation_matrix(PIOVER2 - elat, 'y', unit=u.radian)
```

Comment:
`rotation_matrix` takes `Quantity` as input, so my tendency would be to just use angles directly. If one defines above
```
QUARTER_CIRCLE = 90*u.deg
```
(best to use degrees to avoid multiple unit conversions)
then here it can be
```
@ rotation_matrix(QUARTERCIRCLE - lat, 'y')
@ rotation_matrix(long, 'z')
```

Reply:
Actually, this just moves the unit conversion into `rotation_matrix()`:
```
    if isinstance(angle, u.Quantity):
        angle = angle.to_value(u.radian)
```
but it saves a couple of lines here, so sure.

Reply:
Yes, indeed, I think it helps clarity here1

Reply:
I have made these changes, but will leave these unresolved until the tests work.